api/seeds/shared/set-up-users.js

```diff
@@ -89,7 +89,7 @@ const createUsersToAdd = async (knex, oktaClient) => {
       role_id: stateAdminRoleId,
       status: 'approved',
       updated_by: 'seeds',
-      username: 'em@il.com'
+      username: 'em@il.com',
```

Comment:
idk.  I tend to add them because of my python experience.  It causes me problems though since they are not valid in JSON.

Reply:
@tbolt I think you are getting those errors because you haven't added your cypress.env.json file in the integrationTests folder. I have a copy of the file in keybase